lib/matplotlib/afm.py

```diff
@@ -503,8 +505,8 @@ def get_familyname(self):
 
         # FamilyName not specified so we'll make a guess
         name = self.get_fullname()
-        extras = (br'(?i)([ -](regular|plain|italic|oblique|bold|semibold|'
-                  br'light|ultralight|extra|condensed))+$')
+        extras = (r'(?i)([ -](regular|plain|italic|oblique|bold|semibold|'
```

Comment:
The issue is that what ever font was being used in the OP did not have a `b'FamilyName'` entry, hence hit this code block, where as most afm do have it and return from this function early?

Reply:
Yes. the `re.sub` would always fail. But it's not reached if the entry has a `b'FamilyName'`.